### PR TITLE
Update to java 1.16.0, enable exponential histograms

### PR DESCRIPTION
**Pull request by jack-berg**

### CODE REVIEW

File: src/adservice/Dockerfile [hunk 1]
```diff
@@ -16,6 +16,7 @@ FROM adoptopenjdk/openjdk11
 ENV OTEL_SERVICE_NAME=AdService
 ENV OTEL_LOGS_EXPORTER=otlp
 ENV OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE=DELTA
+ENV OTEL_EXPORTER_OTLP_METRICS_DEFAULT_HISTOGRAM_AGGREGATION=EXPONENTIAL_BUCKET_HISTOGRAM
```
Comment (TylerHelmuth) [inline src/adservice/Dockerfile:19]: Does this mean the AdService will no longer report any explicit bucket histograms?

Comment (jack-berg) [reply src/adservice/Dockerfile:19]: Correct.

Comment (TylerHelmuth) [reply src/adservice/Dockerfile:19]: Does it need to?  Or are there already other services reporting explicit bucket histograms?

Comment (jack-berg) [reply src/adservice/Dockerfile:19]: There are others reporting explicit bucket histograms (frontend and cartservicex).

**Review by TylerHelmuth — APPROVED** [review]

**Pull request merged**